canonical/canonical.com
# djlint --reformat, fix elements nesting

## Done

applied djlint --reformat and fixed element nesting.

## QA

- Check out this feature branch
- Run the site using the command `dotrun`
- View the site locally in your web browser at: `http://0.0.0.0:8002/careers/application/<token>`
- Nothing should have changed in the view
